### PR TITLE
added homepage and filter for desktop

### PR DESCRIPTION
**Pull request by Asuraking1n**

### CODE REVIEW

**Comment by rishichourasia** [comment]
looks good to me as of now ,

**Comment by harshitpaliwal95** [comment]
Looks good

File: html/homepage.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                <a href="#">Login</a>
+                <div class="e-nav-buttons-cons">
+                    <i class="far fa-heart nav-icon" ></i>
+                </div>
+                <div class="e-nav-buttons-cons">
+                    <i class="fas fa-shopping-cart nav-icon"></i>
+                </div>
+            </div>
+        </div>
+        <div class="main-body-sec">
```
Comment (harshitpaliwal95) [inline html/homepage.html:32]: You could use main tag here

Comment (Asuraking1n) [reply html/homepage.html:32]: sure i will fix

**Pull request merged**